res/photon/server.js

```diff
@@ -0,0 +1,27 @@
+// @noflow
+const WebpackDevServer = require('webpack-dev-server');
+const config = require('./webpack.config.js');
+const webpack = require('webpack');
+
+const port = process.env.FX_PROFILER_PORT || 4242;
```

Comment:
should the default be something else (eg 4243) so that we can easily have both dev servers launched together ?
Maybe a different env variable too (eg `FX_PHOTON_PORT`). I'm not defining these env variables myself but somebody could.

Reply:
Different ports work for me.